src/libclipper/src/redis.cpp

```diff
@@ -438,6 +454,22 @@ bool add_application(redox::Redox& redis, const std::string& appname,
   }
 }
 
+bool add_model_links(redox::Redox& redis, const std::string& appname,
```

Comment:
Currently, this method allows users to link an application of input type A to a model of input type B. We should verify that the input types are the same and fail if they aren't.